Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -251,6 +258,17 @@ public override void Analyze(AnalyzeContext context)
                     break;
                 }
 
+                case Validation.PasswordProtected:
```

Comment:
>PasswordProtected [](start = 32, length = 17)

Ouch, we didn't handle this case properly. We should look elsewhere in the source for password-protected scenarios, i.e., why didn't we notice this gap previously?